collate/collate.py

```diff
@@ -197,9 +197,9 @@ def get_columns(self, when=None, prefix=None, format_kwargs=None):
             yield ex.literal_column(column).label(to_sql_name(name))
 
 
-def maybequote(elt):
+def maybequote(elt, quote_choices):
```

Comment:
To me it's not clear based on the name that 'always_quote' being equal to False would mean 'never quote'.